src/app/src/utils/canvas.js

```diff
@@ -6,12 +6,80 @@ export const combineCanvases = (canvasList, direction) => {
     }
 };
 
+function generateWrappedText(context, text, width, font) {
+    context.font = font;
+
+    var x = 20;
```

Comment:
It occurred to me I didn't know the difference between a `var` and `let` declaration. `var` is not scoped, but `let` is.  I didn't see any undesired behavior, but for good measure consider updating `var`s as applicable to `let`.

Reply:
Switched them over to `let` and `const` as applicable